File: Backend/TokanPages.Backend.Shared/Dto/Articles/UpdateArticleDto.cs
using System;

namespace TokanPages.Backend.Shared.Dto.Articles
{
    public class UpdateArticleDto
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string TextToUpload { get; set; }
        public string ImageToUpload { get; set; }
        public bool? IsPublished { get; set; }
        public int? Likes { get; set; }
        public int? ReadCount { get; set; }
    }
}

File: TokanPages/ClientApp/src/Shared/Models/Articles/UpdateArticleDto.ts
interface IUpdateArticleDto 
{
    id: string;
    title: string;
    description: string;
    textToUpload: string;
    imageToUpload: string;
    isPublished: boolean;
    likes: number;
    readCount: number;
}

export type { IUpdateArticleDto }
